app/src/main/java/com/framgia/soundclound/util/BindingUtil.java

```diff
@@ -107,14 +107,15 @@ public static void setImage(ImageView imageView, String urlImage) {
         }
     }
 
+    @BindingAdapter("imgResource")
+    public static void setImage(CircleImageView imageView, String urlImage) {
+            imageView.setImageResource(R.drawable.ic_disk_play);
```

Comment:
ERROR: 'method def' child have incorrect indentation level 12, expected level should be 8.